examples/all-clusters-app/nrfconnect/main/include/AppEvent.h

```diff
@@ -0,0 +1,57 @@
+/*
+ *
+ *    Copyright (c) 2018 Nest Labs, Inc.
```

Comment:
That license header seems to be invalid. It should be the same as in the other files, so `Project CHIP Authors ...`